### PR TITLE
Add type definitions for videojs-hotkeys

### PR DESCRIPTION
**Pull request by Coteh**
https://github.com/ctd1500/videojs-hotkeys

This package is missing definitions which forces TypeScript users to cast videojs player as `any` when referring to the plugin. See https://joeflateau.net/posts/video-js-heart-typescript for more information.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/videojs-hotkeys/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// Type definitions for videojs-hotkeys 0.2
+// Project: https://github.com/ctd1500/videojs-hotkeys
+// Definitions by: James Cote <https://github.com/Coteh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { VideoJsPlayer } from 'video.js';
+
+declare module 'video.js' {
+    interface VideoJsPlayer {
+        hotkeys(options: VideoJsHotkeysOptions): void;
```
Comment (peterblazejewicz) [inline types/videojs-hotkeys/index.d.ts:10]:
```suggestion
        hotkeys(options?: VideoJsHotkeysOptions): void;
```
this look like optional param:
https://github.com/ctd1500/videojs-hotkeys/blob/master/videojs.hotkeys.js#L64

Comment (Coteh) [reply types/videojs-hotkeys/index.d.ts:10]: Think you're right. Default options are used if you don't pass any options. Thanks for pointing this out. :)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Coteh thx!

**Review by amcasey — APPROVED** [review]

**Comment by Coteh** [comment]
Ready to merge

**Pull request merged**